python/console/console_editor.py

```diff
@@ -261,7 +261,10 @@ def contextMenuEvent(self, e):
 
         menu.addSeparator()
         toggle_comment_action = QAction(
-            QgsApplication.getThemeIcon("console/iconCommentEditorConsole.svg"),
+            QgsApplication.getThemeIcon(
```

Comment:
This icon is also used in python/plugins/processing/script/ScriptEditorDialog.py and src/gui/codeeditors/qgscodeeditor.cpp -- they'll also need the palette color set

Reply:
@nyalldawson , done, thanks.